Dockerfile

```diff
@@ -1,13 +1,18 @@
-FROM mhart/alpine-node:10
-RUN apk add --no-cache git bash make gcc g++ python
```

Comment:
Diese Zeilen brauchen wir nicht mehr? Ich kenn mich da nicht sehr gut aus.

Reply:
Ich persönlich finde nichts was dadurch kaputt gegangen ist oder gehen könnte. Weder beim Durchklicken im Frontend noch im Code. Der server-Code macht meinen Erkenntnissen nach nichts zur Laufzeit auf Binary-Ebene, die nicht vorkompilliert irgendwo bereitsteht.

Die Zeile kommt von einem 6 Jahre alten Commit. Im selben Commit und denen davor oder danach kam keine Funktionalität dazu, die irgendwie darauf schließen lässt dass es relevant wäre: https://github.com/BreakOutEvent/breakout-frontend/commit/34d451aa5c5ba4cc0d01ad2a05d496e9cd30a6d9

Der einzige Grund den ich mir denken kann, ist, dass zu der Zeit (Node 8 oder noch älter) ein paar Dependencies noch keine Prebuilt-Binaries hatten und sie im npm-install-Schritt kompilliert wurden. Da das mittlerweile offensichtlich klappt (siehe succeeded PR build), sehe ich keine Notwendigkeit mehr dafür den ganzen Kram mit in das Image zu bauen.

Reply:
Ich denke auch das kann mittlerweile vermutlich raus :) Wenn der build prozess läuft dann geht zur runtime ziemlich sicher nichts kaputt :)